nx-dev/nx-dev/pages/index.tsx

```diff
@@ -139,8 +139,22 @@ export function Index() {
                 </div>
                 <div className="w-full sm:w-auto pt-4 sm:pt-0">
                   <Link href="/conf">
-                    <a className="flex items-center justify-center px-4 py-2 border border-transparent rounded-md shadow-sm text-sm font-medium text-blue-nx-base bg-white">
+                    <a className="flex items-center justify-center px-4 py-2 border border-transparent group rounded-md text-md text-white font-extrabold">
                       Learn more
+                      <span className="group-hover:mr-2">
```

Comment:
This hover on the margin needs to enable a rule that we actually do not need. We can apply the same effect used in the conf page here:
```
transform-gpu transition ease-out duration-200 group-hover:translate-x-2
```
https://github.com/nrwl/nx/blob/master/nx-dev/nx-dev/pages/conf.tsx#L315